docs/UserGuide.md

```diff
@@ -191,15 +191,15 @@ Format: `exit`
 Action | Format, Examples
 --------|------------------
 **List** | `list_student`
-**Find** | `find_student KEYWORD`<br><br>e.g. `find student John`
+**Find** | `find_student KEYWORD`<br><br>e.g. `find_student John`
 **Add** | `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN`<br><br> e.g., `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 l/Sec2 g/95421323 r/Mother`
-**Delete** | `delete_student INDEX`<br><br>e.g. `delete student 3`
+**Delete** | `delete_student INDEX`<br><br>e.g. `delete_student 3`
 
 **Tuition Session**
 
 Action | Format, Examples
 --------|------------------
-**List** | `list session`
-**Find** | `find_student KEYWORD`<br><br>e.g. `find session John`
-**Add** | `add session n/STUDENT_NAME d/DATE t/TIME l/LENGTH_OF_SESSION s/SUBJECT f/FEE`<br><br> e.g., `add session n/John Doe d/14-02-2021 t/1800 l/2 s/Biology f/40`
-**Delete** | `delete session INDEX`<br><br>e.g. `delete session 3`
+**List** | `list_session`
+**Find** | `find_student KEYWORD`<br><br>e.g. `find_session John`
```

Comment:
minor bug: `find_session KEYWORD` instead of `find_student KEYWORD`

Reply:
Waaaa nice catch man! Updated it accordingly already!